.gitignore

```diff
@@ -0,0 +1,2 @@
+*~
+*.pyc
```

Comment:
gitignore is usually put at the repository level, and not in each module, so I've doubts about this. Beside, there is much more complete version in OCA/MQT.